src/routine.js

```diff
@@ -21,6 +21,7 @@ const runner = async (time = 0) => {
   const backoff = parseInt(BACKOFF * (time + 1))
   try {
     await markPastDue(project)
+    await createJournals(project)
```

Comment:
this routine gets executed for both dev avail and dev journal. may need to conditionally run it depending on whether it's the journal one